src/components/modals/DeleteActivitiesModal.svelte

```diff
@@ -0,0 +1,373 @@
+          {#if anchoredActivitiesMap[activityId]?.length}
+            <div class="anchorItem">
+              <div class="directive">
+                <div class="directiveToDelete" class:ignored={!activityDirectivesToDeleteMap[activityId]}>
+                  <div class="directiveName">
+                    <ActivityDirectiveIcon label={`${activityId} - ${$activityDirectivesMap[activityId]?.name}`} />
+                  </div>
+                  <div class="deleteCheckbox">
+                    <label>
+                      <span class="deleteLabel">Delete</span>
+                      <input
+                        type="checkbox"
+                        checked={activityDirectivesToDeleteMap[activityId]}
+                        on:change={() => onToggleDelete(activityId)}
+                      />
+                    </label>
+                  </div>
+                </div>
+                <div class="dependentDirectives" class:ignored={!activityDirectivesToDeleteMap[activityId]}>
+                  {#each anchoredActivitiesMap[activityId] as activityDirective}
+                    <div class="dependentDirective">
+                      <span class="anchorIcon"><ActivityAnchorIconSVG /></span>
+                      <ActivityDirectiveIcon label={`${activityDirective.id} - ${activityDirective.name}`} />
+                    </div>
+                  {/each}
+                </div>
+              </div>
+              <div class="choices">
+                <select
+                  class="st-select"
+                  name={`${activityId}`}
+                  value={activityDirectiveDeletionMap[activityId]}
+                  on:change={onChange}
+                >
+                  <option value={ActivityDeletionAction.ANCHOR_ROOT}>To Nearest Parent</option>
+                  <option value={ActivityDeletionAction.ANCHOR_PLAN}>To Plan Start</option>
+                  <option value={ActivityDeletionAction.DELETE_CHAIN}>Delete Chain</option>
+                </select>
+              </div>
+            </div>
+          {/if}
+        {/each}
+      </div>
+    {/if}
+  </ModalContent>
+  <ModalFooter>
+    <button class="st-button secondary" on:click={() => dispatch('close')}> Cancel </button>
+    <button class="st-button" on:click={confirmDelete}> Confirm </button>
+  </ModalFooter>
+</Modal>
+
+<style>
+  .message {
+    font-weight: 500;
+  }
+
+  .anchorHelp {
+    margin: 1rem 0 0;
+  }
+
+  .anchorHelpToggle {
+    color: #2f80ed;
+    cursor: pointer;
+    margin-bottom: 1rem;
+  }
+
+  .anchorHelpToggle:hover {
+    text-decoration: underline;
+  }
+
+  .helpChoices {
+    display: grid;
+    grid-template-rows: repeat(3, min-content);
+    row-gap: 1rem;
+  }
+
+  .helpChoice {
+    border: 1px solid var(--st-gray-20);
+    border-radius: 5px;
+    column-gap: 5px;
+    display: grid;
+    grid-template-columns: auto min-content;
+    padding: 8px;
+  }
+
+  .helpChoiceTitle,
+  .helpChoiceDescription {
```

Comment:
_Hate_ to ask because you have a lot of classes here but do we care about camel case vs hyphenated style for class names? Seems like most stuff uses hyphens but also it doesn't matter a ton, not sure if it's worth changing these?

Reply:
We should add a stylelint rule for this!

Reply:
Yeah I kinda prefer the hyphenated style more? I might have misremembered a convo we had where we wanted to go camelCase in css though. If that's not the case, I'm down to change it. @AaronPlave @camargo

Reply:
I pushed the hyphenated version